util/gcs/read.go

```diff
@@ -58,7 +58,7 @@ type Build struct {
 }
 
 func (build Build) String() string {
-	return "gs://" + build.Path.String()
+	return build.Path.String()
```

Comment:
I wonder how long it's been part "gs://" has been part of the build path's scheme. Has this function just been returning "gs://gs://path" and some later bit of code has been tolerating it?

Reply:
I believe it is only used for debugging. So something is probably logging gs://gs://foo but this is never actually getting otherwise used.

I suspect that when I initially wrote this the gcs.Path didn't include the gs:// part on its debug String() and then later added it.